pylds/tools.py

```diff
@@ -74,12 +74,23 @@ def draw_lagrangian_descriptor(LD, LD_type, grid_parameters, tau, p_value, norm
         string_title = ''
         print('Incorrect "LD_type". Valid options: forward, backward, total. Plot will appear without title')
     
-    ax.set_title(string_title, fontsize=12)
-    ax.set_xlabel('$x$', fontsize=18)
-    ax.set_ylabel('$y$', fontsize=18)
+    fig.suptitle(string_title, fontsize=14, y=1.04)
+    ax0.set_title('LD values')
+    ax0.set_xlabel('$x$')
+    ax0.set_ylabel('$y$')
+    
+    fig.colorbar(con0, ax=ax0, ticks=np.linspace(np.nanmin(LD),np.nanmax(LD),11),format='%.2f')
+    
+    gx,gy = np.gradient(LD, 0.05, 0.05)
+    scalar = np.sqrt(gx**2 + gy**2)
+    Z = scalar/scalar.max()
+    
+    con1 = ax1.contourf(points_x,points_y,Z,cmap='Reds',levels=200)
+    ax1.set_title('LD gradient magnitude')
+    ax1.set_xlabel('$x$')
+    ax1.label_outer()
+    fig.colorbar(con1, ax=ax1, ticks=np.linspace(np.nanmin(Z),np.nanmax(Z),11),format='%.2f')
```

Comment:
Same comments as for the colorbar of axis `ax0`

Reply:
idem